IPlayer/Player.cs

```diff
@@ -0,0 +1,10 @@
+public class Player
+{
+	int hp {get; set;}
```

Comment:
In C#, we like to use capital letters for public facing variables / methods. Rename to `public int HP {get; set;}`